virttest/utils_net.py

```diff
@@ -1420,8 +1422,8 @@ def set_guest_ip_addr(session, mac, ip_addr,
 
     :return: True if set up guest ip successfully.
     """
+    info_cmd = ""
     try:
```

Comment:
this one no impact.  move this line out of try block is fine, because this line can't raise any exception.